catalog/app/containers/Auth/messages.js

```diff
@@ -79,6 +79,24 @@ export default defineMessages({
     defaultMessage:
       'No Quilt user linked to this OneLogin account. Notify your Quilt administrator.',
   },
+  ssoAzureUse: {
+    id: `${scope}.SSO.Azure.use`,
+    defaultMessage: 'Sign in with Microsoft',
```

Comment:
Note that [Microsoft's branding guidelines](https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-add-branding-in-azure-ad-apps) state that in the end-user UI, it should be called "Microsoft", not "Azure".